examples/text_classification/train.py

```diff
@@ -129,6 +123,8 @@ def test(data_):
                         help='device (default=cpu)')
     parser.add_argument('--data', default='.data',
                         help='data directory (default=.data)')
+    parser.add_argument('--dictionary', default='NONE',
```

Comment:
If you don't provide a default and the user doesn't use --dictionary, args.dictionary will be None. So if you want your default to be None, you don't need to pass in a value for the default keyword, you can just leave it out.